### DISCUSSION TITLE
Audience Targeting for sponsored Product and Brand Ad

### DISCUSSION THREAD

**Discussion by Dipikesh** [amzn/ads-advanced-tools-docs#55] [General]
I want to target audience for sponsored product and brand ads. Audience targeting should be based upon age, gender, location, etc.
How can i achieve this in amazon ads API.

**Comment by elizafarley** [comment]
For both Sponsored Products and Sponsored Brands, only keyword and product targeting are currently supported. Keyword targeting allows you to target based on customer search terms with product targeting allows you to target based on the detail page being viewed. 

Audience targeting is currently only available through Sponsored Display and the Amazon DSP. Hope that helps!

**Reply by Dipikesh** [reply]
Yes, It helped.
Thank you very much
